dwi_ml/experiment/trainor_abstract.py

```diff
@@ -0,0 +1,315 @@
+#!/usr/bin/env python
```

Comment:
Typo in the name of the file?
_trainor_ : _trainer_

Should may be renamed to `tracto*_abstract_trainer.py` or other name where the * means its purpose (e.g. not sure it will be shareable among all tasks/projects)? Not sure whether it should be `*_abstract_trainer.py` or `*_trainer_abstract.py`

Also, this is not a callable script, so the shebang line should not be there.

Reply:
Yes, thanks.

Thanks internet: The confusion in spelling stems from the fact that this word has taken an action and made a noun form for an occupation. Most other words that do this end in –or. For example, supervisor, commentator, and counselor. There is not a specific reason why this word does not conform. One must simply remember that trainer is spelled with an -er.

As for the sharability, we will need many coding sessions to deal with that question, I am aware.

Reply:
tractoML was the initial name I gave to it. I will change to dwi_ml.

Reply:
> Thanks internet: The confusion in spelling stems from the fact that this word has taken an action and made a noun form for an occupation. Most other words that do this end in –or. For example, supervisor, commentator, and counselor. There is not a specific reason why this word does not conform. One must simply remember that trainer is spelled with an -er.

I think `trainer` is the term that has the most widespread adoption within the community.